src/pages/_app.tsx

```diff
@@ -17,7 +21,78 @@ const MyApp: FunctionComponent<AppProps> = ({
     GOVUKFrontend.initAll();
   }, []);
 
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, viewport-fit=cover"
+        />
+        <meta charSet="utf-8" />
+        <meta name="theme-color" content="#0b0c0c" />
+
+        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+
+        <link
+          rel="shortcut icon"
+          sizes="16x16 32x32 48x48"
+          href="/assets/images/favicon.ico"
+          type="image/x-icon"
+        />
+        <link
+          rel="mask-icon"
+          href="/assets/images/govuk-mask-icon.svg"
+          color="#0b0c0c"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/assets/images/govuk-apple-touch-icon-180x180.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="167x167"
+          href="/assets/images/govuk-apple-touch-icon-167x167.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="152x152"
+          href="/assets/images/govuk-apple-touch-icon-152x152.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          href="/assets/images/govuk-apple-touch-icon.png"
+        />
+
+        <meta
+          property="og:image"
+          content="/assets/images/govuk-opengraph-image.png"
+        />
```

Comment:
Looks good, but again could we create a separate component for this here which will mean it's easier to update/test as any changes to the page metadata is isolated in it's own component